bin/main.rb

```diff
@@ -0,0 +1,92 @@
+# rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
+
+# !/usr/bin/env ruby
+
+require '../lib/classes.rb'
+
+puts 'Welcome to our Tic-Tac-Toe Game!'
+
+puts 'Player 1, please type your beautiful name?'
+player_one = Player.new(gets.chomp, 'X')
+puts "Nice name #{player_one.name}!"
+puts 'Players, you are going to be represented by the "X" characters on the board, right? (Press enter to continue)'
+gets.chomp
+
+puts 'Player 2, how could we call this beautiful face?'
+player_two = Player.new(gets.chomp, 'O')
+puts "Huuummmm #{player_two.name}, I was pretty aware that your name would be also beautiful!"
+puts "#{player_two.name}, you are going to be the 'O' character, nice? (Press enter to continue)"
+gets.chomp
+
+puts "#{player_one.name} and #{player_two.name}, swear you won't forget your characters? (Press enter to continue)"
+gets.chomp
+
+puts "Ok #{player_one.name} and #{player_two.name}, lets to the nuts and bolts!"
+puts 'Here is the board of your game, with the respective number for each position, ok?'
+plays = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+def print_board(plays)
+  puts " #{plays[0]} | #{plays[1]} | #{plays[2]}"
+  puts " #{plays[3]} | #{plays[4]} | #{plays[5]}"
+  puts " #{plays[6]} | #{plays[7]} | #{plays[8]}"
+end
```

Comment:
## Add in another class
- The code should respect the single responsibility rule for methods (i.e. each method does exactly one thing). ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/single_responsibility.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))
  - [ ] Add in a Gameboard class